### DISCUSSION TITLE
Nemo - Show selection size after copy or cut

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#114] [Ideas]
If items are selected on Nemo, the bottom status bar correctly shows, for example:

> 2 items selected (2,4 MB (2.386.223 bytes)), Free space:…

If CTRL+C are pressed or copy is selected from the "edit" menu,

> The 2 selected items will be copied if you select the Paste command

Notably, no selection size is shown anymore. It would be more convenient if the size were still shown:

> The 2 selected items  (2,4 MB (2.386.223 bytes)) will be copied if you select the Paste command

The same applies to CTRL+X (cut-and-paste move):

> The 2 selected items  (2,4 MB (2.386.223 bytes)) will be moved if you select the Paste command

It might seem like a minor thing, but many minor things put together make a big difference. I think it would take at most three minutes to implement this, so there is no reason not to do it.

**Comment by rcalixte** [comment]
I liked this idea.

Pull request: linuxmint/nemo#3307
